vscode-extension/src/extension.ts

```diff
@@ -0,0 +1,130 @@
+import { workspace, ExtensionContext, window } from "vscode";
+
+import {
+  CloseAction,
+  ErrorAction,
+  LanguageClient,
+  LanguageClientOptions,
+  RevealOutputChannelOn,
+  ServerOptions,
+} from "vscode-languageclient/node";
+import { findRelayBinary } from "./utils";
+
+let client: LanguageClient;
+
+export async function activate(context: ExtensionContext) {
+  const outputChannel = window.createOutputChannel("Relay Language Server");
+
+  // TODO: Support multi folder workspaces by not using rootPath.
+  // Maybe initialize a client once for each workspace?
+  const relayBinary =
+  // TODO: Use VSCode config instead of process.env
+    process.env.RELAY_BINARY_PATH ?? (await findRelayBinary(workspace.rootPath));
+
+  // TODO: Use VSCode config instead of process.env
+  const outputLevel = process.env.RELAY_LSP_LOG_LEVEL ?? "debug";
+
+  if (!relayBinary) {
+    outputChannel.appendLine(
+      "Could not find relay binary in path. Maybe you're not inside of a project with relay installed."
+    );
+
+    return;
+  }
+
+  outputChannel.appendLine(`Using relay binary: ${relayBinary}`);
+
+  const serverOptions: ServerOptions = {
+    command: relayBinary,
+    args: ["lsp", `--output=${outputLevel}`],
+  };
+
+  // Options to control the language client
+  const clientOptions: LanguageClientOptions = {

```

Comment:
Can we try adding `markdown: {isTrusted: true}`? I'm noticing that hover items are not correctly rendering the content as markdown
<img width="862" alt="Screen Shot 2022-04-04 at 9 01 14 PM" src="https://user-images.githubusercontent.com/162735/161676426-d26d4280-1c63-4964-8e10-37386a3d0719.png">
.

Reply:
okay I think that did the trick. does this look right? 
<img width="482" alt="image" src="https://user-images.githubusercontent.com/6754223/161679221-f08323f3-4133-4176-9936-d2feabf7aebc.png">

Reply:
Perfect